tests/st/utils/data.py

```diff
@@ -395,9 +387,13 @@
     'apiVersion': API_VERSION,
     'kind': 'WorkloadEndpoint',
     'metadata': {
+        'labels': {
```

Comment:
ditto.

and orchestrator should be cni.

Reply:
in fact calicoctl should basically overwrite any value for orchestrator and namespace that you assign and fix them to the values in the values specified in the Metadata and Spec.